sensor_msgs/Image image
sensor_msgs/Image mask
---
std_msgs/String label
